backend/packages/Upgrade/src/api/middlewares/ErrorHandlerMiddleware.ts

```diff
@@ -20,7 +19,7 @@ export class ErrorHandlerMiddleware implements ExpressErrorMiddlewareInterface {
     next: express.NextFunction
   ): Promise<void> {
     // It seems like some decorators handle setting the response (i.e. class-validators)
-    this.log.info('Insert Error in database', error);
+    req.logger.info({ message: 'Insert Error in database', stack: error.stack});
```

Comment:
use logger.error here and pass whole error object. 
req.logger.error(error)

Reply:
ok